### PR TITLE
apply #34932 patch to enable trace module' conv_bn_folding and also enable 3d case

### PR DESCRIPTION
**Pull request by XiaobingSuper**
1.  Making conv_bn_folding can works for traced module.

2. Enable conv3d and BatchNorm3d fusion.

### CODE REVIEW

**Comment by XiaobingSuper** [comment]
@EikanWang , please help merge it.

**Comment by pinzhenx** [comment]
this 3D patch was extract from public pytorch?

**Comment by XiaobingSuper** [comment]
> this 3D patch was taken from public pytorch?

No, public pytorch is not support it now.

**Comment by pinzhenx** [comment]
Are you going to PR them? Because we are not recommended to adding to much stuff to pytorch other than device related code.

**Comment by XiaobingSuper** [comment]
> Are you going to PR them? Because we are not recommended to adding to much stuff to pytorch other than dpcpp device/backend related code.

Yes, I will do it.

**Comment by pinzhenx** [comment]
That’s great

**Comment by EikanWang** [comment]
@XiaobingSuper , Please fix the conflict and i will merge it.

**Comment by XiaobingSuper** [comment]
> @XiaobingSuper , Please fix the conflict and i will merge it.

Fixed.
